### PR TITLE
[connectors] Fix mime types consistency

### PR DESCRIPTION
**Pull request by tdraier**
## Description

Fix mime types consistency - use `.` instead of `-` .

## Risk

None, these types are not used yet

## Deploy Plan

- deploy connectors
- update existing values in data_source_nodes

### CODE REVIEW

File: core/bin/migrations/20241204_create_nodes.rs [hunk 1]
```diff
@@ -29,25 +29,25 @@ enum NodeType {
...
         // Confluence
-        ((|s: &str| s.starts_with("confluence-page-")), "application/vnd.dust.confluence-page"),
+        ((|s: &str| s.starts_with("confluence-page-")), "application/vnd.dust.confluence.page"),
         // Github
-        ((|s: &str| s.starts_with("github-issue-")), "application/vnd.dust.github-issue"),
-        ((|s: &str| s.starts_with("github-discussion-")), "application/vnd.dust.github-discussion"),
-        ((|s: &str| s.starts_with("github-code-")), "application/vnd.dust.github-code-file"),
+        ((|s: &str| s.starts_with("github-issue-")), "application/vnd.dust.github.issue"),
+        ((|s: &str| s.starts_with("github-discussion-")), "application/vnd.dust.github.discussion"),
+        ((|s: &str| s.starts_with("github-code-")), "application/vnd.dust.github.codefile"),
```
Comment (spolu) [inline core/bin/migrations/20241204_create_nodes.rs:36]: code.file I would use

Comment (spolu) [reply core/bin/migrations/20241204_create_nodes.rs:36]: as we will have code.folders soon ?

**Review by spolu — APPROVED** [review]
LGTM

I think code.file is better than codefile as we might have code.folders no?

File: core/bin/migrations/20241204_create_nodes.rs (hunk 1 above)
Comment (tdraier) [reply core/bin/migrations/20241204_create_nodes.rs:36]: For now, it was planned that all folders would be `application/vnd.dust.folder`

**Pull request merged**